### ISSUE TITLE
Fixed flyouts with categories; flyout adjacent to toolbox menu

### ISSUE THREAD

**Issue by tmickel** [LLK/scratch-blocks#256]
Almost all of our potential designs for the future toolbox/flyout combo have something like this:

<img width="208" alt="screen shot 2016-04-20 at 11 33 21 am" src="https://cloud.githubusercontent.com/assets/120403/14680588/cc3d9a4a-06eb-11e6-9e1c-fb42db26ddfa.png">

Where a category menu is visible and the flyout is also visible. Some iterations of the design have the categories to the left of the flyout, vaguely similar to Tickle:

<img width="94" alt="screen shot 2016-04-20 at 11 35 02 am" src="https://cloud.githubusercontent.com/assets/120403/14680638/f487a626-06eb-11e6-9723-38a7e7f5e7f9.png">

We have also done design iterations where the category picker is below the flyout, above the flyout, etc.

ScratchJr:
<img width="314" alt="screen shot 2016-04-20 at 11 36 49 am" src="https://cloud.githubusercontent.com/assets/120403/14680714/36fb1a7e-06ec-11e6-9216-400cb7560665.png">

Basically, we should figure out how to refactor the toolbox/flyout so creating any combination of fixed flyout and positioned category menu is easy. I think we can do this before we have final designs for the toolbox and flyout.

**Comment by tmickel** [comment]
Also closing for now since we have a more focused flyout/toolbox plan.
